proto/interchain_security/ccv/v1/shared_consumer.proto

```diff
@@ -87,16 +87,27 @@ message ConsumerParams {
 message ConsumerGenesisState {
   ConsumerParams params = 1 [ (gogoproto.nullable) = false ];
   ProviderInfo provider = 2 [ (gogoproto.nullable) = false ];
-  // true for new chain, false for chain restart.
-  bool new_chain = 3; // TODO:Check if this is really needed
+  // True for new chain, false for chain restart.
+  // This is needed and always set to true; otherwise, new_chain in the consumer 
+  // genesis state will default to false
+  bool new_chain = 3;
+  // Flag indicating whether the consumer CCV module starts in pre-CCV state
+  bool preCCV = 4;
```

Comment:
A little problematic to understand why there's `new_chain` and `preCCV`, but I get it.

Not: Maybe consider uniform snakecase/camelcase usage

Reply:
preCCV is already a field in the consumer genesis, so I cannot change the name without breaking compatibility.